tests/i2c_end_to_end_sim/src/app_host.xc

```diff
@@ -30,6 +30,7 @@ void host_app(client i2c_master_if i_i2c_host){
   }
   if (version != CONTROL_VERSION) {
     printf("version expected 0x%X, received 0x%X\n", CONTROL_VERSION, version);
+    _Exit(1);
```

Comment:
could you change it with **exit(1)** from stdlib?

Reply:
done in  https://github.com/xmos/lib_device_control/pull/112/commits/e2f7aeb248e7b9516aa9e4d4a0c76195dfeff80f.